sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/UpdateUserFrameworkTest.java

```diff
@@ -157,5 +157,6 @@ private void beforeTest() {
     variable("gradeLevel", System.getenv("sunbird_user_framework_grade_level"));
     variable("medium", System.getenv("sunbird_user_framework_medium"));
     variable("subject", System.getenv("sunbird_user_framework_subject"));
+    variable("id", System.getenv("sunbird_user_framework_id"));
```

Comment:
rename to framework-id

Reply:
add one more test-case - where frame-work id is missing

one more test-case - where framework id is invalid.

Reply:
Done !